### ISSUE TITLE
Suggestion Git installation instructions

### ISSUE THREAD

**Issue by Lourenzutti** [UBC-MDS/UBC-MDS.github.io-jekyll#104]
@joelostblom 

I had to reinstall Ubuntu here. I'm going over MDS install guide. 
For the installation of Git, Ubuntu repository is delayed (it is still on version 2.25). 

I'd suggest to change the command instructions there to 

```
sudo apt-add-repository ppa:git-core/ppa
sudo apt-get update
sudo apt-get install git
```

so it installs the latest version (2.29.2)

**Comment by joelostblom** [comment]
Thanks @Lourenzutti ! Do you know if there is any feature that is especially useful and not available in <=2.25? 

I guess technically all the ubuntu repos are a bit outdated and we would need to use ppa for everything, but I avoided that to not make it to tedious unless I knew of a feature that was needed but was not in the Ubuntu repo version (and there is no snap for git either)

**Comment by Lourenzutti** [comment]
Oh, I see. 

Yeah, I don't think there is anything currently that they will need to use in 2.29 that there isn't in 2.25.  The key was 2.23, like you said in the instructions. 

Sounds good then.  :+1:
